test-swift-docc/test-swift-docc.txt

```diff
@@ -1,8 +1,17 @@
-// Check Swift-DocC can compile and preview documentation.
-//
-// RNU: rm -rf %S/Output
-// RUN: %{docc} convert %S/DocCTest/Sources/DocCTest/DocCTest.docc --output-path %S/Output
-// RUN: ls %S/Output
-
-// REQUIRES: rdar91638258
-
+// Check that Swift-DocC can compile documentation.
+//
+// Clear any existing state:
+// RUN: rm -rf %t.dir
+// RUN: mkdir -p %t.dir
+//
+// Build symbol graphs for the test package:
+// RUN: mkdir %t.dir/symbol-graph-directory
+// RUN: mkdir %t.dir/swiftpm-build-directory
+// RUN: %{swift} build --target DocCTest --package-path %S/DocCTest --build-path %t.dir/swiftpm-build-directory -Xswiftc -emit-symbol-graph -Xswiftc -emit-symbol-graph-dir -Xswiftc %t.dir/symbol-graph-directory
+//
+// Convert the documentation
+// RUN: %{docc} convert %S/DocCTest/Sources/DocCTest/DocCTest.docc --output-path %t.dir/DocCTest.doccarchive --additional-symbol-graph-dir %t.dir/symbol-graph-directory
+//
+// Assert that the produced documentation archive has a 'data' subdirectory as expected
+// RUN: ls %t.dir/DocCTest.doccarchive/data
+// RUN: ls %t.dir/DocCTest.doccarchive/data/documentation/docctest
```

Comment:
Is there any file we can check in here? I imagine this doesn't actually happen, but in general it wouldn't surprise me if a tool created directories even when nothing is actually generated. Or is that "docctest"?

Reply:
That's fair. In general the intention of this test is to be a very basic smoke test to ensure nothing went wrong during a specific toolchain build. We _should_ catch any actual compilation integrity issues with other tests.

But I think asserting on the existence of at least one JSON file still makes sense here- I've added that here: https://github.com/apple/swift-integration-tests/pull/101/commits/2a26855610cbf0387d95c39a90a6eddedac8b7e7.

Thank you!